client/state/happiness-engineers/selectors.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import createSelector from 'lib/create-selector';
+
+function getHappinessEngineersState( state ) {
+	return state.happinessEngineers;
+}
+
+function createHappinessEngineersSelector( fn ) {
+	return createSelector( fn, ( state ) => [ getHappinessEngineersState( state ) ] );
+}
+
+/**
+ * Returns happiness engineers
+ * @returns [] happiness engineers
+ */
+export const getHappinessEngineers = createHappinessEngineersSelector(
+	( state ) => getHappinessEngineersState( state ).items
+);
+
+/**
+ * Returns whether we have loaded some data. Always false when fetching data
+ * @param {{}} state - Previous state
+ * @returns {boolean} - Whether we loaded some data or not
+ */
+export function isRequesting( state ) {
```

Comment:
To avoid naming conflicts, particularly as we consider to global selectors directory (#5954), this should be sufficiently namespaced, e.g.:

```js
export function isRequestingHappinessEngineers( state) {
```